### DISCUSSION TITLE
Can't recieve GitHub emails to verify school email

### DISCUSSION THREAD

**Discussion by ghost** [community/community#146843] [GitHub Education]
### Select Topic Area

Question

### Body

can I verify another way

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Ryler4 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you to the Education category. Good luck!

**Reply by ghost** [reply]
ok thanks !
